LibreNMS/Util/StringHelpers.php

```diff
@@ -194,8 +194,13 @@ public static function hasNumber(string $string): bool
         return (bool) preg_match('/^\s*-?\d+(\.\d+)?/', $string);
     }
 
-    public static function isHex(string $string): bool
+    public static function isHexWithWhitespace(string $string): bool
```

Comment:
I'm not really convinced we want isHexWithWhitespace() .  If we need different delimiters, wouldn't it make sense to match hexToAscii and asciiToHex and have delimiter as an argument?